### PR TITLE
Initial express server Closes #3

### LINKED ISSUE #3: Create Express Server
Create an express server with the following endpoints:
- [x] `GET /wishlist` gets all of the wishlists
- [x] `GET /wishlist/:id` gets a wishlist by id
- [x] `POST /wishlist` creates a new wishlist
- [x] `DELETE /wishlist/:id` deletes a wishlist
- [x] `PUT /wishlist/:id` updates a wishlist

These endpoints won't be functional until we have a database set up.
for now, just set up express so that it will return status codes

### PR DESCRIPTION
**Pull request by thirionlogan**
we had to eject the react app so that we could modify the test config to cover the /server directory

### CODE REVIEW

**Review by brentogunmakin — APPROVED** [review]
Good stuff

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const bodyParser = require('body-parser');
```
Comment (hilldauk) [inline server/app.js:2]: For later, possible use of express.json

**Review by hilldauk — APPROVED** [review]
I like it!!!

File: server/app.js (hunk 1 above)
Comment (thirionlogan) [reply server/app.js:2]: Will put in a later issue

**Pull request merged**